pages/doc/opentelemetry_java_tutorial.md

```diff
@@ -59,17 +59,20 @@ For instrumentation, you use the Java agent provided by OpenTelemetry, which can
     ```
 
 1. Run `./mvnw package` from the root directory of the project.
-1. Download the [OpenTelemetry Java agent](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar).
```

Comment:
Does the numbering have to be fixed here? It looks like all the steps start with 1.